company_website/templates/common_partials/footer.haml

```diff
@@ -82,7 +82,7 @@
             .row
                 .col-lg-9.col-12
                     %span.copyright
-                        © 2019 Code Poets sp. z o.o. All rights reserved.  | NIP: 8952068133 | KRS: 0000598357 | REGON: 363584646
+                        © 2020 Code Poets sp. z o.o. All rights reserved.  | NIP: 8952068133 | KRS: 0000598357 | REGON: 363584646
```

Comment:
You can use here auto updating variable instead of remembering to change it every year. Please use datetime instead of hardcoded value

Reply:
Done